src/local_scheduler/format/local_scheduler.fbs

```diff
@@ -47,6 +47,7 @@ table GetTaskReply {
 table EventLogMessage {
   key: string;
   value: string;
+  time: string;
```

Comment:
I don't see a huge advantage associated with either option, but I can change it if you'd like.